### PR TITLE
Feature/ij 69 team member user index

### PR DESCRIPTION
**Pull request by IeuanSkinner**
I've updated the team member users index view according to the criteria laid out in IJ-69. There are now 3 sections to this page, the first displays some basic info on the total number of users and active (sign in within 30days) users along with a dummy search box whose functionality is to be implemented later in IJ-81.

The second section contains a team members pinned users, this is again a dummy for now as it just takes the first 6 users from the system. The third section shows the remaining users. 

All users are now displayed as basic info cards including their name, contact information and a summary of their last self submitted WBA (if applicable). Currently these WBA are not caring about whether a team member has been granted permission by the user as this is an open question right now, they are also not adding a view log.

Screenshot 2021-03-19 at 16 01 52

If a team member has permission to view a users journal entry and they have not done so then another line is added to the info card outlining that the team member has unread journal entries along with the count.

Screenshot 2021-03-19 at 16 02 07

### CODE REVIEW

**Comment by IeuanSkinner** [comment]
The cards are now grey

Screenshot 2021-03-19 at 16 21 57

File: app/assets/stylesheets/global.scss [hunk 1]
```diff
@@ -13,6 +13,10 @@ body#overboot {
     z-index: 1080;
   }
 
+  .hidden {
+    display: none;
+  }
```
Comment (phil-6) [inline app/assets/stylesheets/global.scss:18]: I'm happy with this but would it be easier to use the bootstrap helper `d-none`

**Review by phil-6 — COMMENTED** [review]
Code all looks good. 
Few questions and thoughts on the UI:

Should the WBA graphs on the user cards be clickable? If so where are they going to link to?
Is the date on the card (underneath their contact details) their release date? Its not clear what this is and I'm not sure we need that on the user card.

Other than that it all looks good!

**Comment by IeuanSkinner** [comment]
> Code all looks good.
> Few questions and thoughts on the UI:
> 
> Should the WBA graphs on the user cards be clickable? If so where are they going to link to?
> Is the date on the card (underneath their contact details) their release date? Its not clear what this is and I'm not sure we need that on the user card.
> 
> Other than that it all looks good!

My assumption is it would lead to /wba_self/:wba_self_id which would give you the full view of the WBA chart including the text next with the wellbeing metrics and score they gave, exactly the same as what the users see  🤔 

It is their release date, I thought this was something that Include UK did want to be able to see? I suppose it's a question for Friday then.

**Comment by bmhtech07** [comment]
Per note in Stand-up, add crisis status to each user

**Comment by IeuanSkinner** [comment]
> Per note in Stand-up, add crisis status to each user

This has now been done:
Screenshot 2021-03-22 at 10 44 23

**Review by AlexW555 — APPROVED** [review]
Really impressive! Love it! The view on mobile looks awesome
Screenshot 2021-03-22 at 4 40 05 pm

**Pull request merged**